tests/disaster-recovery/sc_arbiter/test_shutdowns.py

```diff
@@ -0,0 +1,487 @@
+            time.sleep(delay * 60)
+
+        if immediate:
+            end_time = datetime.now(timezone.utc)
+            sc_obj.post_failure_checks(
+                start_time, end_time, wait_for_read_completion=False
+            )
+
+        sc_obj.cephfs_logreader_job.delete()
+        log.info(sc_obj.cephfs_logreader_pods)
+        for pod in sc_obj.cephfs_logreader_pods:
+            pod.wait_for_pod_delete(timeout=120)
+        log.info("All old CephFS logreader pods are deleted")
+
+        # check for any data loss
+        assert sc_obj.check_for_data_loss(
+            constants.LOGWRITER_CEPHFS_LABEL
+        ), "[CephFS] Data is lost"
+        log.info("[CephFS] No data loss is seen")
+        assert sc_obj.check_for_data_loss(
+            constants.LOGWRITER_RBD_LABEL
+        ), "[RBD] Data is lost"
+        log.info("[RBD] No data loss is seen")
+
+        # check for data corruption
+        pvc = get_pvc_objs(
+            pvc_names=[
+                sc_obj.cephfs_logwriter_dep.get()["spec"]["template"]["spec"][
+                    "volumes"
+                ][0]["persistentVolumeClaim"]["claimName"]
+            ],
+            namespace=constants.STRETCH_CLUSTER_NAMESPACE,
+        )[0]
+        logreader_workload_factory(
+            pvc=pvc, logreader_path=constants.LOGWRITER_CEPHFS_READER, duration=5
+        )
+        sc_obj.get_logwriter_reader_pods(constants.LOGREADER_CEPHFS_LABEL)
+
+        wait_for_pods_to_be_in_statuses(
+            expected_statuses=constants.STATUS_COMPLETED,
+            pod_names=[pod.name for pod in sc_obj.cephfs_logreader_pods],
+            timeout=900,
+            namespace=constants.STRETCH_CLUSTER_NAMESPACE,
+        )
+        log.info("[CephFS] Logreader job pods have reached 'Completed' state!")
+
+        assert sc_obj.check_for_data_corruption(
+            label=constants.LOGREADER_CEPHFS_LABEL
+        ), "Data is corrupted for cephFS workloads"
+        log.info("No data corruption is seen in CephFS workloads")
+
+        assert sc_obj.check_for_data_corruption(
+            label=constants.LOGWRITER_RBD_LABEL
+        ), "Data is corrupted for RBD workloads"
+        log.info("No data corruption is seen in RBD workloads")
+
+        # check the connection scores if its clean
+        mon_conn_score_map = {}
+        mon_pods = get_mon_pods()
+        for pod in mon_pods:
+            mon_conn_score_map[get_mon_pod_id(pod)] = fetch_connection_scores_for_mon(
+                pod
+            )
+        log.info("Fetched connection scores for all the mons!!")
+        mon_quorum_ranks = get_mon_quorum_ranks()
+        log.info(f"Current mon_quorum ranks : {mon_quorum_ranks}")
+
+        # check the connection score if it's clean
+        sc_obj.validate_conn_score(mon_conn_score_map, mon_quorum_ranks)
+
+    @pytest.mark.parametrize(
+        argnames="iteration, delay",
+        argvalues=[
+            pytest.param(2, 5),
+        ],
+    )
+    def test_zone_crashes(
```

Comment:
The steps in this test look almost the same as the previous shutdown test, with the only difference being the way you induce the node failure. Perhaps you could handle both of these scenarios in a single test function using parameterization. This approach would result in cleaner code and eliminate redundancy